cucumber-messages/go/messages.proto

```diff
@@ -585,13 +609,15 @@ message SupportCodeConfig {
   repeated ParameterTypeConfig parameterTypeConfigs = 4;
 }
 
+// TODO: Rename to Hook
 message TestCaseHookDefinitionConfig {
   string id = 1;
   string tagExpression = 2;
   // TODO: rename to sourceReference as Location is another type and this is ambiguous.
   SourceReference location = 3;
 }
 
+// TODO: Rename to StepDefinition
```

Comment:
I thought its useful to have Config as a suffix as the language probably has a StepDefinition class to encompass the pattern and code provided by the user